lib/app_state/app_either.dart

```diff
@@ -0,0 +1,11 @@
+import 'package:dartz/dartz.dart';
+import 'package:fluffychat/app_state/failure.dart';
+import 'package:fluffychat/app_state/success.dart';
+import 'package:flutter/foundation.dart';
+
+typedef AppEither = Either<Failure, Success>;
+typedef AppEitherNotifier = ValueNotifier<AppEither>;
```

Comment:
Not a fan of it. I know that will take a lot of code space but keep `Either` is more readable

Reply:
it can avoid dev use different Success type
eg: Either<Failure, GetContactsSuccess>

Reply:
Use by accepted convention by team is enough.

Reply:
fixed